data/NotificationEntry.css

```
@@ -23,3 +23,28 @@
     color: mix(@theme_fg_color, @error_color, 0.6);
     font-weight: 600;
 }
+
+.close {
+    padding: 3px;
+    border: none;
+    border-radius: 100%;
+    background-image:
+        linear-gradient(
+            to bottom,
+            @BLACK_300,
+            @BLACK_500
+        );
+    box-shadow:
+        inset 0 0 0 1px alpha (#fff, 0.02),
+        inset 0 1px 0 0 alpha (#fff, 0.07),
+        inset 0 -1px 0 0 alpha (#fff, 0.01),
+        0 0 0 1px alpha (#000, 0.7),
+        0 1px 2px alpha (#000, 0.16),
+        0 2px 3px alpha (#000, 0.23);
+    margin: 3px;
+}
+
+.close-image {
```

Probably don't need to add a new class here. You could select with 
```suggestion
.close image {
```

---

When I did this, exactly as you suggested, something else blocked me from coloring the icon, having it always be the fg color.

---

It worked now. :confused: